### DISCUSSION TITLE
How Do I Use the Bootstrap Accordion classes/capability?

### DISCUSSION THREAD

**Discussion by snesbittsea** [razonyang/hugo-theme-bootstrap#837] [Q&A]
I have a content section that would benefit from leveraging the Bootstrap Accordion capability. The question is how?

I've tried adding and index.md file with the Bootstrap Accordion example, but it appears that those classes are not seen - at least no content in an class=accordion" div is displayed.

What am I missing and how can I get it to work?

Thank in advance for your help and patience.

-steve

**Comment by razonyang** [comment]
Hi, I think the raw HTML was omitted by Markdown parser, you can change it by setting the following configuration on `config.toml`.

```toml
[markup.goldmark.renderer]
unsafe = true
```

However, the recommended way to do that is creating a shortcode, such as `layouts/shortcodes/my-accordion.html`.

```html
// PUT YOUR ACCORDION CODE HERE.
```

And then use the shortcode in Markdown pages.

```markdown
{{< my-accordion >}}
```

> The `unsafe` setting is not need to be set.

You can also define a general shortcode that accepts headers and their contents parameters, the details can be found at https://gohugo.io/templates/shortcode-templates/.

**Comment by snesbittsea** [comment]
Hi Razon!



Thanks much for the quick reply and excellent suggestions!



Was able to get it working with minimal difficulty using shortcodes.



I'm pretty new to Go and Hugo, so I'm struggling to figure out what the questions are!



-steve






---- On Sat, 24 Dec 2022 19:59:09 -0700 Razon Yang ***@***.***> wrote ---





Hi, I think the raw HTML was omitted by Markdown parser, you can change it by setting the following configuration on config.toml.

[markup.goldmark.renderer]
unsafe = true


However, the recommended way to do that is creating a shortcode, such as layouts/shortcodes/my-accordion.html.

// PUT YOUR ACCORDION CODE HERE.


And then use the shortcode in Markdown pages.

{{< my-accordion >}}


The unsafe setting is not need to be set.


You can also define a general shortcode that accepts headers and their contents parameters, the details can be found at https://gohugo.io/templates/shortcode-templates/.

**Reply by razonyang** [reply]
My pleasure.
